operator/internal/pkg/custom/verify.go

```diff
@@ -25,6 +25,11 @@ func (r *Resource) DefaultFn(context.Context, *unstructured.Unstructured, *logr.
 func (r *Resource) CheckFn(ctx context.Context, manifestObj *unstructured.Unstructured, logger *logr.Logger,
 	clusterInfo custom.ClusterInfo,
 ) (bool, error) {
+	// if manifest resource is in deleting state - validate check
+	if !manifestObj.GetDeletionTimestamp().IsZero() {
```

Comment:
Anther issue regarding this `CheckFn`, it's been used with 3 different case, `VerifyResources`, ` Install`, `uninstall`, for uninstalling, return true if `mainfestObj` is in deleting state make sense, but for ` Install` and `VerifyResources` is this determine valid?

Reply:
Yes for `Install` and` VerifyResources` we need to check if the custom resource is in ready state. 
This check ONLY happens if the necessary flag is set.